### ISSUE TITLE
Issue when upgrading from 1.3.1 to 1.4.0

### ISSUE THREAD

**Issue by kodiakbrown** [docbook/xslTNG#100]
Hello,

First, thank you for the Docbook XSL stylesheets!  They are great.

I've been using 1.3.1 to chunk my docbook file to html files.  All has been working fine.  I downloaded 1.4.0 today and ran through my scripts without changing anything and started getting this message:

`java : Error at char 5 in expression in xsl:when/@test on line 93 column 37 of docbook.xsl:
 ...
XPDY0050  The root of the tree containing the context item is not a document node
  In template rule with match="node()" on line 88 of docbook.xsl
     invoked by xsl:apply-templates at file:///C:/docbook/docbook-xslTNG-1.4.0/xslt/docbook.xsl#68
     invoked by xsl:for-each at file:///C:/docbook/docbook-xslTNG-1.4.0/xslt/docbook.xsl#66
  In template rule with match="/" on line 37 of docbook.xsl
The root of the tree containing the context item is not a document node`

Without knowing all too much of what to do about it, I looked at the difference in the docbook.xsl file between 1.3.1 and 1.4.0.  I commented out lines 93 to 95.

`<xsl:when test="not(/h:html)">
 <xsl:sequence select="."/>
 </xsl:when>`

Then I re-ran my scripts and everything is working fine now.   This could be an issue on my end or with my docbook XML file, but I figured I'd post it here as an issue only because everything was working fine when I was using 1.3.1.

Thank you again for the style sheets!

**Comment by ndw** [comment]
Is there any chance you can provide a sample document that demonstrates this?

**Comment by kodiakbrown** [comment]
[KeywordExample.txt](https://github.com/docbook/xslTNG/files/6401039/KeywordExample.txt)


I've attached a sample file that causes the error in the original post.

I created the sample file to post a second issue but like this issue, it is something that is probably not configured correctly on my end.  The 2nd issue I wanted to post is that the keywordset is only getting transformed to a meta tag when the keywordset is under the root book or article element.  If I add keywords to my chapters or sections that get chunked, the chapter/section keywords do not get transformed into the relative chunked output file.  Only the keywordset from the root element gets output to the chapter/section chunks.

For the sample file, I am seeing 
`<meta name="keywords" content="KEYWORD1,KEYWORD2"/>`
in CHAPTER1.html, SECTION1.html, SECTION2.html.  I would expect to see this in index.html, but the remaining chunks would have the keywords defined directly within their respective element within the source file.

I didn't mean to stack two separate issues into one thread, but they are both using the same sample file (attached here).

Thank you!
